src/firmware/app/control/physbot.c

```diff
@@ -16,8 +16,8 @@ PhysBot app_physbot_create(const FirmwareRobot_t *robot, float *destination,
         .disp = dot2D(major_vec, dr), .vel = dot2D(major_vec, v), .accel = 0, .time = 0};
     const Component minor = {
         .disp = dot2D(minor_vec, dr), .vel = dot2D(minor_vec, v), .accel = 0, .time = 0};
-    const Component rot = {.disp = min_angle_delta(
-                               app_firmware_robot_getOrientation(robot), destination[2])};
+    const Component rot = {.disp =
+                               destination[2] - app_firmware_robot_getOrientation(robot)};
```

Comment:
this is so that the robot will spin, i.e. if destination is 510deg and current orientation is 180deg then it will spin counterclockwise, rather than clockwise to 510-360=150deg